com.github.vikdevelop.penpot.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: com.github.vikdevelop.penpot
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+command: penpot.sh
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-download
+  - --device=dri
+build-options:
+  append-path: /usr/lib/sdk/node16/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: penpot
+    buildsystem: simple
+    build-options:
+        env:
+          XDG_CACHE_HOME: /run/build/penpot/flatpak-node/cache
+    build-commands:
+    - install -D -t /app/bin penpot.sh
+    - install -D -t /app/share/applications flatpak/com.github.vikdevelop.penpot.desktop
+    - install -D -t /app/share/metainfo flatpak/com.github.vikdevelop.penpot.metainfo.xml
+    - install -D -t /app/share/icons/hicolor/128x128/apps flatpak/icons/com.github.vikdevelop.penpot.png
+    sources:
+      - type: archive
+        url: https://github.com/vikdevelop/penpot/archive/refs/tags/0.2.tar.gz
+        sha256: e9d35f51d3238d3c472e8d0d94904e17f329097f1c1996262257b8b54310c5d3
+      - type: script
+        dest-filename: penpot.sh
+        commands:
+          - zypak-wrapper.sh /app/main/penpot "$@"
+  - name: build
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/main
+      - cp -R * /app/main
+    sources:
+      - type: archive
+        url: https://github.com/vikdevelop/penpot/releases/download/0.2/penpot-build-flatpak-v0.2.tar.gz
```

Comment:
what the difference with this and the one above?

Reply:
@hfiguiere  This is the archive where the application is built, because `yarn --offline` didn't work in `flatpak-builder` (I used `generated-sources.json` generated by `flatpak-yarn-generator.py`) so I added it to `build-options` in the manifest on the local drive on my computer:
```yaml
build-options:
    build-args:
      - --share=network
```
and the application was built, and started successfully, so I compressed `$FLATPAK_DEST/main` into a tar archive and uploaded it on GitHub. So I tried to build it locally using flatpak-builder and the build was successful and the application started. Manifest, that works on local HW is [available on Github repository vikdevelop/penpot](https://github.com/vikdevelop/penpot/blob/main/manifest.yaml).

Reply:
it still doesn't explain why you'd need two tarballs.

Reply:
also that mean you copy x86_64 binaries on aarcch64.

Reply:
and you seem to copy two versions of `electron-v*-linux-x64.zip` which represent 156MB

Reply:
In first archive is the source code from GitHub, in second archive is main directory, and in that directory is the content built by the `electron packager`. So from the first archive, it installs the .desktop file, metainfo file, icon, etc., and from the second archive, it installs the files that electron packager built that are required to run the application. 

Why did I create a second tarball, and not upload it to GitHub? 
Because it's an unnecessarily large size, so I found it easier to compress it and upload it as a second tarball as attachment of GitHub release 0.2.